terraform.tf

```diff
@@ -57,7 +57,10 @@ module "ecs" {
   ecs_service_task_port                 = "${var.ecs_service_task_port}"
   ecs_service_task_definition_file_path = "${var.ecs_service_task_definition_file_path}"
   ecs_import_schools_task_definition_file_path = "${var.ecs_import_schools_task_definition_file_path}"
+  ecs_vacancies_scrape_task_definition_file_path = "${var.ecs_vacancies_scrape_task_definition_file_path}"
   import_schools_entrypoint              = "${var.import_schools_entrypoint}"
+  vacancies_scrape_entrypoint            = "${var.vacancies_scrape_entrypoint}"
+  vacancies_scrape_schedule_expression   = "${var.vacancies_scrape_schedule_expression}"
```

Comment:
This feels like a useful value to include in our config file `staging.tfvars.example` so that it continues to be as straight forward as possible to tweak this value.

Reply:
This is in the staging.tfvars.example file now